hudi-common/src/main/java/org/apache/hudi/common/table/timeline/HoodieInstant.java

```diff
@@ -226,6 +275,8 @@ public int compareTo(HoodieInstant o) {
 
   @Override
   public String toString() {
-    return "[" + ((isInflight() || isRequested()) ? "==>" : "") + timestamp + "__" + action + "__" + state + "]";
+    return "[" + ((isInflight() || isRequested()) ? "==>" : "")
+        + timestamp + "__" + action + "__" + state
+        + (StringUtils.isNullOrEmpty(stateTransitionTime) ? "" : ("__" + stateTransitionTime)) + "]";
   }
```

Comment:
Guess the check `StringUtils.isNullOrEmpty` is unnecessary because `stateTransitionTime` is by defaut an empty string.

Reply:
I changed `stateTransitionTime` default to null to keep the consistent behavior in `HoodieArchivedMetaEntry`